packages/cli/src/commands/spec.ts

```
@@ -81,4 +225,117 @@ export const specCommand = new Command('spec')
+      .filter(line => line.trim().match(/^As\s+a/i));
+    
+    stories.forEach((story, index) => {
+      // Parse "As a X, I want Y, so that Z"
+      const match = story.match(/As\s+a[n]?\s+(\w+),?\s*I\s+want\s+(.+?),?\s*so\s+that\s+(.+)/i);
+      if (match) {
+        scenarios.push({
+          name: `User ${match[1]} completes goal`,
+          given: `I am a ${match[1]}`,
+          when: `I want to ${match[2]}`,
+          then: `I can ${match[3]}`
+        });
```

Biome (2.3.14)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)